### DISCUSSION TITLE
[General] npx zksync-cli dev start can't work

### DISCUSSION THREAD

**Discussion by mikoto10001** [zkSync-Community-Hub/zksync-developers#473] [General] [answered]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement.

I want to migrate my contracts from L1 to zksync. Currently, I want to test it locally, so I follow the tutorials to start a dev net locally but didn't work.

### What is the specific issue or error you're encountering?

`npx zksync-cli dev start` didn't work and stack at `load metadata for docker.io/library/ubuntu:latest` or `postgres、geth、zksync`，I have tried to use a proxy, but it can't work too. The memory node and persist mode all facing the same problem.

### Can you share the error messages or logs you're receiving, if any?

![error1](https://github.com/zkSync-Community-Hub/zksync-developers/assets/73458266/22e5cc48-8cbf-4021-a490-396e265abfab)
![error2](https://github.com/zkSync-Community-Hub/zksync-developers/assets/73458266/934e396e-276f-4db7-9a47-7d4a119c51e6)

### Have you made any recent changes to the contract before encountering this issue?

no

### Are there any external libraries or contracts that your contract interacts with?

no

### Can you provide the relevant portions of your contract code where the issue is occurring?

no

### Have you tried to isolate the problem, and if so, what were the results?

no

### What steps have you already taken to try to resolve the issue?

I have tried to use a proxy but it doesn't work.

**Comment by JackHamer09** [comment, marked as answer]
@mikoto10001, the screenshots you provided indicate an unusually long waiting time. This suggests that you might be encountering issues such as:
- Connectivity Issue: Please check your internet connection. Consider disabling your VPN, or it's possible that your region is being blocked by Docker services.
- Resource Constraints: Ensure that your system has sufficient CPU, RAM, and storage space available.
